test/attachments.js

```diff
@@ -20,7 +20,7 @@ describe('entries with attachments', () => {
   test('should return attachment data', () => {
     return expect(
       couch.getAttachmentByName('entryWithAttachment', 'test.txt', 'b@b.com')
-    ).resolves.toEqual(Buffer.from('THIS IS A TEST'));
+    ).resolves.toBe('THIS IS A TEST');
```

Comment:
This is probably a breaking change but unlikely to have an impact. I checked our uses of getAttachment and none of them assume that the value is a buffer